controllers/genres.js

```diff
@@ -1,25 +1,29 @@
 const models = require('../models')
 
 const getAllGenres = async (request, response) => {
-  const genres = await models.Genres.findAll()
+  try {
+    const genres = await models.Genres.findAll()
 
-  return response.send(genres)
+    return response.send(genres)
+  } catch (error) {
+    return response.sendStatus(500)
+  }
 }
 
 const getGenresByIdWithAuthorsAndNovels = async (request, response) => {
-  const { id } = request.params
+  try {
+    const { id } = request.params
```

Comment:
You changed the variable in your route in `index.js` to `identifier` to match the others but did not change it here. There is no parameter called `id` so this is trying to do a `where: { id: undefined }` which causes the route to throw an error. Make sure you are testing everything before you submit.